src/components/CardStats.jsx

```diff
@@ -25,15 +29,15 @@ const CardStats = ({
       <p>Giver(s)</p>
     </div>
 
-    <div className="col-4 text-center">
+    <div className="col-4 text-center" style={ maxAmount ? {display: 'flex', alignItems: 'center', justifyContent: 'center'} : {}}>
```

Comment:
can you create a class and just apply the class here, instead of the css

Reply:
I'll do that but it still needs a condition though